vulkan_generator/pylintrc

```diff
@@ -279,9 +279,9 @@ generated-members=
 
 [FORMAT]
 
-# Melih : We deviate from Google Standard here and using line-lenght 100 instead of 80
+# Melih : We deviate from Google Standard here and using line-lenght 120 instead of 80
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=120
```

Comment:
this file should not be part of this PR. we can create a separate PR for this.

Reply:
Agreed. Will remove from final submission.